src/components/Create/Create.tsx

```diff
@@ -128,23 +122,16 @@ export function Create() {
             }
             description={
               <Trans>
-                Reward contributors with NFTs when they meet your funding
-                criteria.
+                Reward your supporters with one or more custom NFT tiers.
```

Comment:
I think we could pare this back more

```suggestion
                Reward your supporters with custom NFTs.
```